src/index.js

```diff
@@ -1,42 +1,87 @@
 import "./styles.css";
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
 import { getNextGameboard, isFieldValueAlive } from "./gameboard";
 
-function renderGameboard(gameboard) {
-  return `<div class="gameboard">${gameboard
-    .map(
-      (row) =>
-        `<div class="gameboard-row">${row
-          .map(
-            (field) =>
-              `<div class="gameboard-field"><div class="gameboard-cell gameboard-cell--${
-                isFieldValueAlive(field) ? "alive" : "dead"
-              }"></div></div>`
-          )
-          .join("")}</div>`
-    )
-    .join("")}</div>`;
+// function renderGameboard(gameboard) {
+//   return `<div class="gameboard">${gameboard
+//     .map(
+//       (row) =>
+//         `<div class="gameboard-row">${row
+//           .map(
+//             (field) =>
+//               `<div class=""><div class="gameboard-cell gameboard-cell--${
+//                 isFieldValueAlive(field) ? "alive" : "dead"
+//               }"></div></div>`
+//           )
+//           .join("")}</div>`
+//     )
+//     .join("")}</div>`;
+// }
+
+function Gameboard({ gameboard }) {
+  return (
+    <div className="gameboard">
```

Comment:
Consider using verifiers wether gameboard is an array. Otherwise map might result in an error if changes are made. Consider using PropTypes (React property verification).